stories/dropdown.stories.js

```diff
@@ -91,6 +99,105 @@ type Dropdown: (
                         title="Sample"
                     />
                 </div>
+                <div style={dropdownContainer}>
+                    <p>A regular dropdown without anything</p>
```

Comment:
```suggestion
                    <p>A regular dropdown without ?</p>
```

Not sure what "without anything" means, but maybe it's clear to everyone else. 🙂